### ISSUE TITLE
Feature: add logo, colors in twig call

### ISSUE THREAD

**Issue by tacman** [endroid/qr-code-bundle#66]
I'd like to be able to set the logo and other options when making the twig call, and I don't think it's possible right now.

The way LiipImagineBundle allows run-time parameters is to pass them as an argument in the twig call, |image_filter('widen', {width: 430}), so I was wonder if there was some way to do that in the QR code generator, e.g.

```twig
<img src="{{ qr_code_url('My QR Code', {logo: '/assets/symfony.png') }}" />

OR 

<img src="{{ qr_code_url('My QR Code', bgcolor='pink', logo='my.svg') }}" />

```

Alternatively, can you document how to create a QR code with a dynamic logo?

Thanks for this great bundle and library.

**Comment by skrilax91** [comment]
It could be great to also add possibility to dynamically change the label text

**Comment by epitre** [comment]
PR #69 is taking care of that :)

**Comment by endroid** [comment]
Implemented in 4.2.1.
